docs/osf-trivia-list/dts.md

```diff
@@ -0,0 +1,53 @@
+# Dasharo Tools Suite
+
+## How to get Dasharo Tools Suite Supporters Edition?
+
+DTS SE is availabele only to Dasharo Supporters for personal use. Commercial
+use of DTS SE should be discussed directly with [3mdeb](mailto:leads@3mdeb.com)
+or [Dasharo Team](mailto:contact@dasharo.com).
+
+In general there are three ways to support us:
+
+### I have more time then money and I can code
+
+Please help us develop Dasharo influence on open-source firmware market and
+spread the word about it. There are multiple ways to do that:
+
+* [Join Dasharo Matrix
+  Community](../../ways-you-can-help-us/#join-dasharo-matrix-community) and
+  support other members of community
+* [Join Dasharo open-source firmware
+  vPub](../../ways-you-can-help-us/#join-dasharo-open-source-firmware-vpub) or
+  other related event organized by 3mdeb or Dasharo Team.
+* [Write a Google review](../../ways-you-can-help-us/#write-a-google-review)
+* [Follow us on social
+  media](../../ways-you-can-help-us/#follow-us-on-social-media) and help
+  spreading the word about Dasharo.
+* [Contribute](../../ways-you-can-help-us/#contribute-through-github)
+  documentation, test restuls, [Dasharo Hardoware Compatibility List
+  Reports](../glossary/#dasharo-hardware-compatibility-list-report)
+
+To get access to DTS SE you should [contact 3mdeb](mailto:leads@3mdeb.com) or
+[Dasharo Team](mailto:contact@dasharo.com) and prove your contribution
+regarding above areas of support and influence. If it would be meaningful we
+would be glad to give you access to DTS SE for a year.
+
+### I have more time then money and I can't code
+
+Please [contribute](../../ways-you-can-help-us/#contribute-through-github)
+features, review and bug fixes to our repositories.
```

Comment:
All of this when `I can't code` ?

Reply:
It would be up to the team judgement if whatever was done is significant enough to provide access. I think our expectations about spreading the word are quite high and I would like to keep it that way.

Reply:
Section `I can't code` mentions that you can contribute and review features and bug fixes. If somoeone cannot code, can they contribute new feature or review code?

Reply:
Improved https://github.com/Dasharo/docs/pull/310/commits/096e3ac8f427857ba73f9a31809645d91e961996